vimeo-networking/src/main/java/com/vimeo/networking/model/Subscriptions.java

```diff
@@ -0,0 +1,108 @@
+package com.vimeo.networking.model;
+
+import com.vimeo.stag.GsonAdapterKey;
+
+import org.jetbrains.annotations.NotNull;
+
+import java.io.Serializable;
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * A model that holds the type of push subscriptions a user has. This class
+ * has both getters and setters since a user can update their subscriptions
+ * using a Patch.
+ * <p>
+ * Created by zetterstromk on 12/15/16.
+ */
+public class Subscriptions implements Serializable {
+
+    private static final long serialVersionUID = 3088065484753327987L;
+
+    public static final String KEY_COMMENT = "comment";
+    public static final String KEY_CREDIT = "credit";
+    public static final String KEY_LIKE = "like";
+    public static final String KEY_REPLY = "reply";
+    public static final String KEY_FOLLOW = "follow";
+    public static final String KEY_VIDEO_AVAILABLE = "video_available";
+
+    @GsonAdapterKey(KEY_COMMENT)
+    boolean mComment;
+
+    @GsonAdapterKey(KEY_CREDIT)
+    boolean mCredit;
+
+    @GsonAdapterKey(KEY_LIKE)
+    boolean mLike;
+
+    @GsonAdapterKey(KEY_REPLY)
+    boolean mReply;
+
+    @GsonAdapterKey(KEY_FOLLOW)
+    boolean mFollow;
+
+    @GsonAdapterKey(KEY_VIDEO_AVAILABLE)
+    boolean mVideoAvailable;
+
+    public boolean isComment() {
```

Comment:
In looking at this again, I realized that the naming could be changed a little to more accurately represent what the methods do. How about this pattern: 
```
public void receiveThing(boolean receive)
public boolean isReceivingThing() 
```

For example: 

```
public void receiveLikes(boolean receive)
public boolean isReceivingLikes()
```